integrations/lever/nango.yaml

```diff
@@ -26,6 +26,111 @@ integrations:
+            get-archive-reasons:
+                description: |
+                    Get all Posts for your account in Lever
+                output: SuccessResponse
+                endpoint:
+                    method: GET
+                    path: /archived/reasons
+                    group: Opportunities
+            posting:
+                description: |
+                    Get single posts for your account in Lever
+                output: SuccessResponse
+                input: SinglePost
+                endpoint:
+                    method: GET
+                    path: /posts
+                    group: Posts
+            update-opportunity-links:
+                description: |
+                    Update the links in an opportunity
+                output: SuccessResponse
+                input: UpdateLinks
+                endpoint:
+                    method: POST
+                    path: /links
+                    group: Opportunities
+            update-opportunity-sources:
+                description: |
+                    Update the sources in an opportunity
+                output: SuccessResponse
+                input: UpdateSources
+                endpoint:
+                    method: POST
+                    path: /sources
+                    group: Opportunities
+            update-opportunity-stage:
+                description: |
+                    Update the stage in an opportunity
+                output: SuccessResponse
+                input: UpdateOpportunityStage
+                endpoint:
+                    method: POST
+                    path: /stages
+                    group: Opportunities
+            update-opportunity-tags:
+                description: |
+                    Update the tags in an opportunity
+                output: SuccessResponse
+                input: UpdateTags
+                endpoint:
+                    method: POST
+                    path: /tags
+                    group: Opportunities
+            update-opportunity-archived:
+                description: |
+                    Update the tags in an opportunity
+                output: SuccessResponse
+                input: ArchiveOpportunity
+                endpoint:
+                    method: PUT
+                    path: /archived
+                    group: Opportunities
+            apply-posting:
+                description: |
+                    submit an application on behalf of a candidate. his endpoint can only be used to submit applications to published or unlisted postings
+                output: SuccessResponse
+                input: ApplyToPosting
+                endpoint:
+                    method: POST
+                    path: /posts/apply
+                    group: Posts
+            update-opportunity:
+                description: |
+                    Update an opportunity
+                output: object[] | ReturnObjUpdateOpportunity
```

Comment:
Can the output be more specific? `object[]` isn't a good enough output for a user IMO